UAS.Client/src/components/TheHeader.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="header">
+    <div class="header__hamburger-wrapper">
+      <button class="hamburger__button">
+        <div class="hamburger__section"></div>
+        <div class="hamburger__section"></div>
+        <div class="hamburger__section"></div>
+      </button>
+    </div>
+    <div class="header__logo-wrapper"><span class="bold">ULTRALIGA</span> ANALYST SERVICE</div>
+    <div class="header__seasons-wrapper">
+      <select name="seasons" id="seasons" class="seasons__select">
+        <option value="1-PO" selected>1-PO</option>
+      </select>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+scss
```

Comment:
What is that

Reply:
What do you mean dude

Reply:
line 22, should it be there?

Reply:
Of course not :cry:. I was confused because You did not pointed out exact mistake, like in second requested change. Have a good day :beers:.

Reply:
The changes came into force in abdf3662248f1781cb7dc610b339b6412a0d2734.